SPECS/qemu-kvm/qemu-kvm.spec

```diff
@@ -1,7 +1,7 @@
 Summary:	QEMU is a machine emulator and virtualizer
 Name:		qemu-kvm
 Version:	4.2.0
```

Comment:
Nit: since you're already here, could you replace all tabs with spaces and align these fields?